modules/knative-serving-advanced-config.adoc

```diff
@@ -6,7 +6,8 @@
 [id="knative-serving-advanced-config_{context}"]
 = Advanced configuration options
 
-This section provides information about advanced configuration options for Knative Serving.
+You can install Knative Serving by using the default settings, or configure more advanced settings in the `KnativeServing` custom resource (CR). Configuring these advanced settings allows you to modify functionality such as use of certificates for your Knative Serving deployment, and the number of high availability replicas that are available.
+// needs an update once more options are documented here
```

Comment:
Is it not documented yet though - can you add a Jira please to document configuring it using the KnativeServing CR as a follow up?

Reply:
Yes only things that can be configured downstream, however feel free to open Jira issues for things that are not documented yet since I would like to add to this section